# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/util-linux/hotfix-blkzone.patch.avr32
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- util-linux-2.36.2/sys-utils/blkzone.c.vanilla	2021-10-07 00:30:23.712134047 +0200
+++ util-linux-2.36.2/sys-utils/blkzone.c	2021-10-07 00:30:36.342133543 +0200
@@ -260,10 +260,10 @@
  * blk_zone_report hasn't been packed since https://github.com/torvalds/linux/commit/b3e7e7d2d668de0102264302a4d10dd9d4438a42
  * was merged. See https://github.com/karelzak/util-linux/issues/1083
  */
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
 			const struct blk_zone *entry = &zi->zones[i];
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
 			unsigned int type = entry->type;
 			uint64_t start = entry->start;
 			uint64_t wp = entry->wp;
